### PR TITLE
[New Rule] Attempt to Disable Auditd Service

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Adversaries may attempt to disable the Auditd service in an attempt to evade detection. Auditd is a Linux service that provides system auditing and logging. Disabling the Auditd service can prevent the system from logging important security events, which can be used to detect malicious activity.

## Telemetry
This rule is created as `low` severity, as it can be administrator activity as well. 

{43DB81AA-4238-4607-8A80-F6CCE54620E8}

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: New - Guidelines

These guidelines serve as a reminder set of considerations when proposing a new rule.

### Documentation and Context

- [ ] Detailed description of the rule.
- [ ] List any new fields required in ECS/data sources.
- [ ] Link related issues or PRs.
- [ ] Include references.

### Rule Metadata Checks

- [ ] `creation_date` matches the date of creation PR initially merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive, considering performance for diverse environments. Non ecs fields should be added to `non-ecs-schema.json` if not available in an integration.
- [ ] `min_stack_comments` and `min_stack_version` should be included if the rule is only compatible starting from a specific stack version.
- [ ] `index` pattern should be neither too specific nor too vague, ensuring it accurately matches the relevant data stream (e.g., use logs-endpoint.process-* for process data).
- [ ] `integration` should align with the `index`. If the integration is newly introduced, ensure the manifest, schemas, and `new_rule.yaml` template are updated.
- [ ] `setup` should include the necessary steps to configure the integration.
- [ ] `note` should include any additional information (e.g. Triage and analysis investigation guides, timeline templates).
- [ ] `tags` should be relevant to the threat and align/added to the `EXPECTED_RULE_TAGS` in the definitions.py file.
- [ ] `threat`, `techniques`, and `subtechniques` should map to ATT&CK always if possible.

#### New BBR Rules
- [ ] `building_block_type` should be included if the rule is a building block and the rule should be located in the `rules_building_block` folder.
- [ ] `bypass_bbr_timing` should be included if adding custom lookback timing to the rule.

### Testing and Validation

- [ ] Provide evidence of testing and detecting the expected threat.
- [ ] Check for existence of coverage to prevent duplication.

**Review by Samirbous — APPROVED** [review]

File: rules/linux/defense_evasion_attempt_to_disable_auditd_service.toml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "Use Case: Threat Detection",
+    "Tactic: Defense Evasion",
+    "Data Source: Elastic Defend",
+    "Data Source: Elastic Endgame",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event") and (
+  (process.name == "service" and process.args == "stop") or
```
Comment (shashank-elastic) [inline rules/linux/defense_evasion_attempt_to_disable_auditd_service.toml:59]: A very similar was deprecated in the past https://github.com/elastic/detection-rules/pull/2163

FYI - this turns to be very noisy! :)

Comment (Aegrah) [reply rules/linux/defense_evasion_attempt_to_disable_auditd_service.toml:59]: This is specifically for Auditd. Is that on the list you provided? I only see one for firewalls; which will be far more common.

File: rules/linux/defense_evasion_attempt_to_disable_auditd_service.toml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+[metadata]
+creation_date = "2024/08/28"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/08/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may attempt to disable the Auditd service in an attempt to evade detection. Auditd is a Linux service that
```
Comment (Mikaayenson) [inline rules/linux/defense_evasion_attempt_to_disable_auditd_service.toml:10]:
```suggestion
Adversaries may attempt to disable the Auditd service to evade detection. Auditd is a Linux service that
```

**Review by Mikaayenson — APPROVED** [review]

**Review by imays11 — APPROVED** [review]

**Pull request merged**